src/AzureDevOpsDemoBuilder/Program.cs

```diff
@@ -12,13 +13,21 @@ namespace AzureDevOpsDemoBuilder
 {
     public class Program
     {
+        //public static IConfiguration Configuration { get; } = new ConfigurationBuilder()
```

Comment:
If the changes are only commented-out lines, this file doesn't need to be in the commit :)

Reply:
I added these lines to read the keys from local AppSettings.
Later I configured keys in Azure App Configuration, so just commented to see whether it reads from Azure App Config. 

it can able to read Azure App config, i'll remove these lines of code.

Reply:
Ok. So should we merge then come back to it? Or can we deploy to another location to test first?

Reply:
I tested this with PR release itself in another environment, it is working. But I'll remove these commented lines and do another push. Later I'll merge.